content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js

```diff
@@ -300,6 +309,28 @@
                 dataLayer.push(uploadPayload);
             }
         }
+
+        /**
+         * Returns the tab item index that corresponds to the deep link in the URL fragment
+         *
+         * @private
+         * @returns {Number} the tab item index if it exists, -1 otherwise
+         */
+        function getDeepLinkItemIdx() {
```

Comment:
This is very similar to `getDeepLinkItem`. Could we create just one helper method for getting the item, that is shared between the two usages?

Reply:
@vladbailescu It's been addressed in https://github.com/adobe/aem-core-wcm-components/pull/1011/commits/3422ecd9c7dfc2b21784e2261cfe3636b5e76395